src/main/java/META-INF/MANIFEST.MF

```diff
@@ -0,0 +1,3 @@
+Manifest-Version: 1.0
```

Comment:
There is a file `MANIFEST.MF` already present in `src/main/resources` no need to create it again